courses/free-courses-en.md

```diff
@@ -885,6 +885,7 @@
 * [Learn to Program: The Fundamentals](https://www.coursera.org/learn/learn-to-program) (Coursera)
 * [Practical Python: An Immersive Python Course](https://practical.learnpython.dev) - Nina Zakharenko
 * [Practical Python Programming](https://dabeaz-course.github.io/practical-python/) - David Beazley
+* [Problem Solving, Python Programming, and Video Games](https://www.coursera.org/learn/problem-solving-programming-video-games) - Duane Szafron (Coursera)
```

Comment:
is this a time-limited course that we have to remove in 6 months? if so we can't add it.

Reply:
No, its not a time-limited course and one can attempt it over a longer time period than just 6 months by resetting the deadlines (which actually only apply for paid courses)